testdata/filters/credentials.txt

```diff
@@ -0,0 +1 @@
+{ "credentials" : ["admin:admin", "admin2:admin2", "admin3:admin3", "admin4:admin4"]}
```

Comment:
It might be ok to use a file for testing purpose, but is it possible to load credentials from k8s secret in cluster?

Reply:
okok I'm going to research on that!